crates/db_views/src/structs.rs

```diff
@@ -184,3 +185,11 @@ pub struct CustomEmojiView {
   pub custom_emoji: CustomEmoji,
   pub keywords: Vec<CustomEmojiKeyword>,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// An external auth view.
+pub struct ExternalAuthView {
+  pub external_auth: ExternalAuth,
+}
```

Comment:
This view seems totally pointless if it only has one field. Why not move the methods into `crates/db_schema`?